batch_gpu_script.py

```diff
@@ -0,0 +1,10 @@
+import hailtop.batch as hb
```

Comment:
This isn't used by the PR, is it?  Looks like it was a script you used for testing and has your hardcoded paths embedded.